src/Layer/OGC3DTilesLayer.js

```diff
@@ -22,19 +22,14 @@ import PointsMaterial, {
 } from 'Renderer/PointsMaterial';
 
 const _raycaster = new THREE.Raycaster();
-
+const viewers = [];
```

Comment:
Can you add a comment saying that it stores ids of viewers where tilesets have been added and explaining why and how it is related to the cache please ? You can also link the issue I think